javafx/build.gradle

```diff
@@ -43,15 +43,35 @@ dependencies {
 }
 
 task unpackSource(type: Copy) {
-    dependsOn configurations.source
-    doFirst {
-        assert openJfxVersion.getProperty('sha256') == sha256(configurations.source.singleFile)
+    if (!project.hasProperty("openjfxPath")) {
```

Comment:
Can we only do the hasProperty check in 1 place and do the right thing there? Checking 3 places seems like a lot. patchJfx can just depend on unpackSource which can then check the property and make the right choice.

Reply:
Make sense. Fixed in next revision.